ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/recovery/RecoveryMonitorTest.java

```diff
@@ -66,8 +65,8 @@ public void testRecoveryMonitorWithFailure() throws Exception {
         recoveryEnvironmentBean.setRecoveryListener(true); // configure the RecoveryMonitor
 
         RecoveryManager manager = RecoveryManager.manager(RecoveryManager.DIRECT_MANAGEMENT);
-
-        manager.addModule(new XARecoveryModule()); // we only need to test the XARecoveryModule
```

Comment:
We have a policy of only making changes that are relevant to the issue being fixed. This change seems unrelated.

Reply:
Actually, change is related because as I have introduce new class variable in `XARecoveryModule` class to maintain recovery state, therefore in order to access the value of recovery state in test, I needed XARecoveryModule reference.

Reply:
Ah okay, apologies for not checking